components/List/StyledList.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export const StyledList = styled.li`
```

Comment:
I would suggest to rename this to e.g. "StyledListItems" because right now the name sounds more like an "ul" than "li". Also maybe you reconsider doing some of the styling for the whole ul instead for each li?

Reply:
That's a good suggestion, I refactored accordingly.

Reply:
changed accordingly